extensions/azurecore/src/account-provider/providerSettings.ts

```diff
@@ -81,6 +82,11 @@ const publicAzureSettings: ProviderSettings = {
 				endpointSuffix: '.core.windows.net',
 				azureResourceId: AzureResource.AzureStorage
 			},
+			azureKustoResource: {
```

Comment:
It would be great to have this added for the other clouds as well. Or is this something that only applies to the public Azure cloud?

Reply:
Yes, it only applies to public Azure clouds for now. Justin is going to enable gov clouds but that's for later